styles/index.css

```diff
@@ -0,0 +1,171 @@
+@import url(/styles/doc.css);
+@import url(/styles/avatar.css);
+
+/* CSS Reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* Root */
+:root {
+  --primary-color: #000;
+  --secondary-color: #fff;
+  --background-color: #fff;
+  --logo-font-family: "Lato", sans-serif;
+  font-size: 62.5%;
+}
+
+/* Elements */
+header {
+  background-color: var(--primary-color);
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  background-color: var(--background-color);
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--primary-color);
+}
+
+/* Utility Classses */
+
+.mg-0-2 {
```

Comment:
You should design system and have value in terms of 8 or 4 , also you can name your classes margin-vertical  and margin horizontol.

Reply:
Changed margin utility classes as per tailwind